content/news/events-august-2025.md

```diff
@@ -0,0 +1,44 @@
++++
+headline = "ARTICLE"
+title = "Foulab Events - August 2025"
+author_name = "tlot"
+lastmod = "2025-08-04" 
++++
+
+{{< centered-image src="events-august-2025.png" width="384">}}
+
+<center><i>Credit zero-eks-five-cee</i></center>
+
+<br/>
+
+August already? Time sure flies when you are having FOULAB. Let's have more - here is how to do it:
+
+### No Net Night - Tuesday August 5th @ 18:30 - 23:30
+
+No WAN, all LAN. Come hang out and explore the foulab internal network as well as various curiosities around the lab without internet access.
+
+<br/>
+
+### Foulab Monthly Meeting and Election Kickoff - Tuesday August 12th @ 19:30 - 21:30
+
+Do-orcracy turns into dem-ocracy for one day while we hold not only our monthly meeting but also kickoff the elections for the board!
+
+<br/>
+
+### ESPHome workshop - Saturday August 30th @ 13:00 - 18:00
+
+ESPHome is an open-source firmware framework that lets you turn your esp32 based project into reality and integrate it in home assistant.
+
+It is aimed at beginner users that want to have a first hand experience with it and get started on doing cool projects that are accessible to non programmers.
+
+The workshop will show: installing ESPHome in home assistant, setting the wifi variables and connecting a esp32-c3 to a bme280 sensor(temperature, humidity and pressure) and writing the firmware using ESPHome yaml config file.
+
+-Hardware kit is 20$, donation to Foulab to cover expenses. Includes esp32c3, bme280, jumper cables and it's yours to keep after the workshop.
```

Comment:
Was this intended to render as a bulleted list? It doesn't show up that way on the site: https://foulab.org/news/events-august-2025/